docs/docs/topics/ppl.md

```diff
@@ -165,6 +166,35 @@ allow:
         before: 4:30PM
 ```
 
+## Device Matcher
+
+A device matcher is an object with operators as keys. It supports the following operators:
+
+- `is` - an exact match of the device ID.
+- `approved` - true if the device has been approved. This is an enterprise-only feature.
+- `type` - Specifies the type of device to match on. The available types are `enclave_only` and `any`.
```

Comment:
@calebdoxsey I think we need more context around what counts as "enclave" versus not between common types like TPM modules, FIDO keys, etc.